variable-fonts-experiments/readme.md

```diff
@@ -0,0 +1,25 @@
+# Project Title
+
+You can read the full article at  
+[Web Fundamentals](https://developers.google.com/web/fundamentals/design-and-ux/typography/variable-fonts/). 
+
+
+This is a collection of experiements inspired by [axis-praxis.org](https://www.axis-praxis.org/specimens/__DEFAULT__) and [typetools.typenetwork.com/](https://typetools.typenetwork.com/typespec/). The goal is to demonstrate the flexability of Variable Fonts and the potential they hold for designers and developers.
+
+
+## Getting Started
+
+These are standalone HTML files and can be run in a browser. 
+
+Working Demo be seen at [https://variable-font-experiments.glitch.me/](variable-font-experiments.glitch.me/)
```

Comment:
You can use https://googlechrome.github.io/samples/variable-fonts-experiments/variable-fonts-viewer/viewer.html as well instead of glitch.

Reply:
I dont mind, wasnt sure how the viewer worked. But feel free to move things around.